lib/node_loader.js

```diff
@@ -18,9 +18,37 @@ util.clientSideMonitoring = {
   Publisher: require('./publisher').Publisher,
   configProvider: require('./publisher/configuration'),
 };
+var iniLoader = require('./shared-ini').iniLoader;
 util.iniLoader = require('./shared-ini').iniLoader;
```

Comment:
Extra lines?

Reply:
Removed in [`c34f9aa` (#3853)](https://github.com/aws/aws-sdk-js/pull/3853/commits/c34f9aa7825aabe1ec9e66a4271e504e9ff29368)